### PR TITLE
missing envs bug fix

### PR DESCRIPTION
**Pull request by alexabramov-nice**

### CODE REVIEW

File: bpjs-debugger/src/main/java/il/ac/bgu/se/bp/execution/BPJsDebuggerImpl.java [hunk 1]
```diff
@@ -277,12 +277,14 @@ public BooleanResponse nextSync() {
     }
 
     private boolean runNextSync() {
+        logger.info("runNextSync state: {0}", state.getDebuggerState());
+
+        if(!isThereAnyPossibleEvents()){
+            nextSyncOnNoPossibleEvents();
+        }
```
Comment (alexabramov-nice) [inline bpjs-debugger/src/main/java/il/ac/bgu/se/bp/execution/BPJsDebuggerImpl.java:282]: no return?

Comment (tallitman) [reply bpjs-debugger/src/main/java/il/ac/bgu/se/bp/execution/BPJsDebuggerImpl.java:282]: No, inside 'nextSyncOnNoPossibleEvents' we are waiting for some external event to be inserted, and then he will be inside the 'selectableEvents' set we will get in line 287.

**Comment by alexabramov-nice** [comment]
not merging yet, adding ITs to this branch

**Comment by alexabramov-nice** [comment]
* fixed stop process
* fixed CLI debugger stop process
* step into, out, over ITs

**Comment by alexabramov-nice** [comment]
image

**Comment by alexabramov-nice** [comment]
added program started/stopped indication

**Pull request merged**